crt_portal/static/js/word_count.js

```diff
@@ -7,21 +8,21 @@ var wordCountArea = document.getElementById("word_count_area");
 // Show word count area for JS-enabled browsers:
 if (wordCountArea) { wordCountArea.removeAttribute('hidden'); }
 
-function updateWordCount (e) {
+function updateWordCount (e, max='', textAreaElem='') {
```

Comment:
Setting `''` as the default for `max` will ask the code in this function to do computations like `500 - ''` and `if (words >= '')`. If we don't have a max value supplied, we can't update word count, so I think throwing a JS error would be an improvement on setting a default value here.